common/build-helper/texmf.sh

```diff
@@ -0,0 +1,25 @@
+# Avoid repeating same thing accross a gazillion files
+
+# rsync isn't needed for everything but it's far easier to just put it here
+hostmakedepends+=" unzip xz rsync"
+# python_version isn't needed for everything either
+python_version=3
+
+do_install() {
```

Comment:
Helpers are meant to be composable with other helpers and build styles. Change to build style or make it function called from template's do_install.

If build style, extraction can work with create_wrksrc set by default, dafault do_extract and moving files in hook.

Reply:
Hm, ok, I think I might change it to a build-style, should simplify the code somewhat further.

Reply:
Moved to build-style in latest push.